### DISCUSSION TITLE
Bed leveling on S1 Pro (F4 SoC)

### DISCUSSION THREAD

**Discussion by fpmirabile** [mriscoc/Ender3V2S1#438] [Q&A]
Hello everyone!

I setup my S1 pro and change the display and updated with this firmware.
Now, testing I realize that there is something odd with the leveling, I cannot get a print done correctly on the first layer as I used to (verified all values from the configuration).

I can't tell exactly what is going wrong but looks like the leveler is not working? Values looks very high and the Z index looks to change very often, different from what used to happen before with the stock firmware.

Is there something I can do to fix it? 

Btw, I am using the latest release version for the F4 chip.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/issues/195

**Reply by fpmirabile** [reply]
Is there a fix planned or do you know the reason?
I got to use mine without editing the mesh manually. Changed the Z Fade to 10mm and also did a deviation test (but I don't think this changes anything)

**Reply by mriscoc** [reply]
Currently the only solution is to correct the mesh points manually.

**Comment by csanii123** [comment]
I was having a problem on my Ender 3 v2 using Manual Mesh leveling.....I followed the steps to do the mesh, saved the mesh, added the M420 S1 code to my file after the G28 line but I couldn't get it to work. I figured I either missed a step or didn't calibrate properly so I tried over and over. Finally on attempt #5 I found that in the Mesh Leveling menu I hadn't enabled "Activate Leveling". As soon as I did that the bed level test came out perfect.
